docs/UserGuide.md

```diff
@@ -1,30 +1,38 @@
----
-layout: page
-title: User Guide
----
+# User Guide
 
-NUS Classes is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, NUS Classes can get your contact management tasks done faster than traditional GUI apps.
+NUS Classes is a desktop app for NUS Computing professors to manage their tasks and contacts. It includes task management features such as 
+adding and editing tasks, finding tasks by date or name, tagging tasks with appropriate tags and marking them as done. It also includes contact management features
+such as finding contacts, assigning contacts to specific tasks and tagging contacts. 
+
+Tasks that are overdue and not done are displayed in red, tasks due within a day are displayed in yellow
+and tasks with more than a day are displayed in yellow. Tasks also contain a clickable zoom link, for easy access to lectures/tutorials or zoom meetings.
```

Comment:
NUS Classes also provides a simple alert feature for tasks by displaying tasks in different color based on the urgency of the task. Tasks that are overdue are marked as red, whereas, tasks that are nearing deadline is marked as yellow.